spec/features/service_subsidy_spec.rb

```diff
@@ -56,7 +56,7 @@
       wait_for_javascript_to_finish
       adjusted_pi_contribution = @direct_cost - (@direct_cost * percent_subsidy)
       pi_field_value = find('#pi_contribution').value.gsub(/,/, '')
-      expect(pi_field_value).to eq('$' + adjusted_pi_contribution.to_s)
+      expect(pi_field_value).to eq('$' + adjusted_pi_contribution.to_s + '0')
```

Comment:
And why are you adding  '0' string to this?  I would think the adjusted_pi_contribution calculation would be correct.  This seems hacky to me.